assets/app/features/upgrade/controllers/upgrade-nodes.controller.js

```diff
@@ -0,0 +1,20 @@
+(function() {
+    'use strict';
+
+    /**
+    * @ngdoc function
+    * @name crowbarApp.controller:UpgradeNodesController
+    * @description
+    * # UpgradeNodesController
+    * This is the controller used on the Stop OpenStack Services page
```

Comment:
This is the controller used on the Upgrade Nodes step, not Stop OpenStack Services